src/components/InventoryGroupDetail/GroupTabDetails.js

```diff
@@ -50,56 +51,37 @@ const GroupTabDetailsWrapper = ({
     [groupId, groupName]
   );
 
-  const [component, setComponent] = useState(conventionalSystemsContent);
-
-  useEffect(() => {
-    if (activeTab === hybridInventoryTabKeys.conventional.key) {
-      setComponent(conventionalSystemsContent);
-    } else {
-      setComponent(immutableSystemsContent);
-    }
-  }, [activeTab]);
-  const handleTabClick = (_event, tabIndex) => {
-    setTab(tabIndex);
-    if (tabIndex === hybridInventoryTabKeys.conventional.key) {
-      setComponent(conventionalSystemsContent);
-    } else {
-      setComponent(immutableSystemsContent);
-    }
-  };
-
-  const [activeTabKey, setActiveTabKey] = useState(0);
-
   return (
     <Tabs
-      activeKey={activeTabKey}
-      onSelect={(event, value) => setActiveTabKey(value)}
+      activeKey={activeTab}
+      onSelect={(_event, value) => setActiveTab(value)}
       aria-label="Group tabs"
       role="region"
       inset={{ default: 'insetMd' }} // add extra space before the first tab (according to mocks)
       mountOnEnter
       unmountOnExit
     >
-      <Tab eventKey={0} title="Systems" aria-label="Group systems tab">
+      <Tab eventKey="systems" title="Systems" aria-label="Group systems tab">
         <PageSection>
           {canViewHosts && hasEdgeImages ? (
             <Tabs
               className="pf-m-light pf-v5-c-table"
-              activeKey={activeTab && tab == 0 ? activeTab : tab}
-              onSelect={handleTabClick}
+              activeKey={activeSystemsTab}
+              onSelect={(_event, tabIndex) => setActiveSystemsTab(tabIndex)}
               aria-label="Hybrid inventory tabs"
+              unmountOnExit
```

Comment:
This makes the trick and fixes the bug. Otherwise, more InventoryTable's are mounted (even though not visible) and talk to the same redux store which caused conflicts.

Reply:
Nice catch! Maybe let's also get the components out of the `useMemo`s. They really don't need to be in there.

Reply:
Actually, I suspect that if they are out of the `useMemo` You'll also not need the `unmountOnExit`.

Reply:
I have removed useMemo's, though this unmountOnExit is better kept for now. Without it, I have noticed two instances of InventoryTable rendered for both conventional and immutable tabs. The table rows even more weird when I tried to remove this parameter: the conventional table had rows from immutable tab systems table which again has showed that the settings from `entities` are being used by both table instances. Let's try to keep at maximum one inventory table rendered on any view. We should later revisit the old PR where we tried to isolate stores.